plugins/cpp/parser/src/cppparser.cpp

```diff
@@ -358,6 +484,32 @@ bool CppParser::parse()
   return success;
 }
 
+void CppParser::initBuildActions()
+{
+  (util::OdbTransaction(_ctx.db))([&, this] {
```

Comment:
Vexing parse parens -> uniform, again.

Reply:
See above.

Reply:
Fixed in 4301dea623f06401b259435c3999d35a3f646dd6.